apis/README.md

```diff
@@ -0,0 +1,6 @@
+# APIs development guidelines
+

```

Comment:
Add a rationale section as well where we can explain some of the decisions we took yesterday, e.g. organization of folders, when/how to share, the folder layout (e.g. $(version)/components/schema ) ... 
And some of the conventions (even better if the latter are all checked in the tests)